src/util/Parameters.h

```diff
@@ -141,6 +141,13 @@ struct dbl {
 struct szt {
   size_t operator()(const auto& s) const { return std::stoull(s); }
 };
+struct bl {
+  bool operator()(const auto& s) const {
+    if (s == "true") return 1;
+    if (s == "false") return 0;
+    return (bool)std::stoi(s);
```

Comment:
Please return `true` and `false` instead of `0` and `1`.

And this will convert `227349` to `true` in the casting from `stoi`, which is probably not whhat we want.
Maybe just expect the strings "true" or "false" and otherwise throw an exception like `the string value for a bool parameter must either be "true" or "false"` or something like that.